disabled-tests-inspector/src/main/java/io/quarkus/ts/GitHubAnalysisResource.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.ts;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.jboss.logging.Logger;
+
+@ApplicationScoped
+public class GitHubAnalysisResource {
+
+    private static final Logger LOG = Logger.getLogger(GitHubAnalysisResource.class);
+
+    @Inject
+    DisabledTestAnalyserService analyserService;
+
+    public void startAnalysis() {
+        String repoOwner = System.getProperty("repoOwner", "org");
+        String repoName = System.getProperty("repoName", "repo-name");
+        String branches = System.getProperty("branches", "main");
+        String baseOutputFileName = System.getProperty("baseOutputFileName", "disabled-tests-report.json");
```

Comment:
This is Quarkus application, out of principle I don't see a reason why to stick with one config source. You could just get these from the config. Not important though.